### PR TITLE
Implement mixpanel for `crm/contacts`

### PR DESCRIPTION
**Pull request by freaz**
## Description

It is a bit controversial, because Mixpanel isn't actually CRM and doesn't allow creating new contacts (users).
You as developer need to send unique id of the user, and if it doesn't exists it will be created.

Also Mixpanel doesn't does not expose users filtering in public API.

It share provider from https://github.com/superfaceai/station/pull/158

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: superface/super.json [hunk 1]
```diff
@@ -297,12 +297,21 @@
         },
         "hubspot": {
           "file": "../capabilities/crm/contacts/maps/hubspot.suma"
+        },
+        "mixpanel": {
+          "file": "../capabilities/crm/contacts/maps/mixpanel.suma"
         }
-      }
+      },
+      "priority": [
```
Comment (Jakub-Vacek) [inline superface/super.json:305]: Maybe remove priority 🤷

**Review by Jakub-Vacek — APPROVED** [review]

File: superface/super.json (hunk 1 above)
Comment (freaz) [reply superface/super.json:305]: Shouldn’t be priority added by CLI only of there is more than one provoder configured? 

What it changes if it is set?

How sdk behaves when concrete provoder is passed in runtime? Does it failover anyway?

Comment (Jakub-Vacek) [reply superface/super.json:305]: I will look into CLI behavior.

It changes nothing, it's just dead code. You can keep it, my OCD kicked in 😅

Comment (freaz) [reply superface/super.json:305]: Ok, I am curious as it is what users will have in super.json as I am not expecting, they will be removing it.

Comment (freaz) [reply superface/super.json:305]: Hmm when I went to fix it, I noticed there is more providers. When I specify priority and omit some providers does it meant fallback to them won't happen?

For sure this is under documented.

**Pull request merged**